src/ImageSharp/Formats/Tiff/Compression/Decompressors/CcittTwoDimensionalCodeType.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Formats.Tiff.Compression.Decompressors
+{
+    internal enum CcittTwoDimensionalCodeType
+    {
+        None = 0,
```

Comment:
It would be nice to have enumerations like this documented even if they're internal.